### ISSUE TITLE
deno DoH: Blocking not working

### ISSUE THREAD

**Issue by amithm7** [serverless-dns/serverless-dns#51]
```sh
doh -k facebook.com https://dns.rethinkdns.localhost:8080/1:4AcHAP__________-f8\=
```
should be blocked but isn't.

**Comment by amithm7** [comment]
Something here is losing the config / url?

https://github.com/serverless-dns/serverless-dns/blob/33500e993279bf1dd7fa58d9458ccbfae3e8a614/src/server-deno.ts#L106-L110

**Comment by ignoramous** [comment]
The "blocklist" `Map` was always adjudged "empty" on Deno, because this line of code:

https://github.com/serverless-dns/serverless-dns/blob/1b60d566a72100d53a7e0af3005047bcc5298cde/src/commons/util.js#L315

Seems like `__proto__` isn't Deno's cup of tea...

**Comment by ignoramous** [comment]
Blocks should be up and running on Deno now: https://github.com/serverless-dns/serverless-dns/commit/881d89125cd1c0a8fb524c235d1ed8ebe0f61607

**Comment by amithm7** [comment]
@ignoramous  yes, blocking works, but node/ fly is broken with that commit: https://github.com/serverless-dns/serverless-dns/runs/4924780290?check_suite_focus=true

**Comment by ignoramous** [comment]
Fixed: https://github.com/serverless-dns/serverless-dns/commit/15f5df62aa826bbbf12ed2f9e88d3e0863bd66e5 Thanks. :D
